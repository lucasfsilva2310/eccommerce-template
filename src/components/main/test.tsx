import { render, screen } from '@testing-library/react'

import Main from './index'

describe('Main Component', () => {
  it('Should render Main component', () => {
    // Render component
    const { container } = render(<Main />)

    // Get the rendered component
    const component = screen.getByRole('heading', { name: /Hello world/i })

    // Check if component exists
    expect(component).toBeInTheDocument()

    // Match snapshot
    expect(container.firstChild).toMatchSnapshot()
  })
})
