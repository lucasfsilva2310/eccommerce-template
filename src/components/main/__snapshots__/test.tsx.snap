// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Component Should render Main component 1`] = `
<main
  class="bg-slate-500"
>
  <h1>
    Hello World
  </h1>
</main>
`;
